README.md

```diff
@@ -39,12 +39,12 @@ Without installing go, kind can be built reproducibly with docker using `make bu
 Stable binaries are also available on the [releases] page. Stable releases are
 generally recommended for CI usage in particular.
 To install, download the binary for your platform from "Assets" and place this
-into your `$PATH`. E.G. for macOS:
+into your `$PATH`:
 
 ```console
-curl -Lo ./kind-darwin-amd64 https://github.com/kubernetes-sigs/kind/releases/download/v0.4.0/kind-darwin-amd64
-chmod +x ./kind-darwin-amd64
-mv ./kind-darwin-amd64 /some-dir-in-your-PATH/kind
+curl -Lo ./kind-$(uname)-amd64 https://github.com/kubernetes-sigs/kind/releases/download/v0.4.0/kind-$(uname)-amd64
```

Comment:
`uname` prints `Linux` on my machine, we could store a variable with lowercased uname output though, either with `tr` or `awk`

Reply:
unless you are saying that github will download this with the wrong case?

we should also probably save this as just `./kind` and let the commands below not need the platform at all.

Reply:
The uppercase download will work on github. Of course the cleaner solution would be using `tr`. I just wanted to keep it simple though.

And yes just using `./kind` is a good idea. Let me amend those changes.